BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudPlayerState.cs

```diff
@@ -394,5 +394,150 @@ public BrainCloudPlayerState(BrainCloudClient client)
+        /// </param>
+        public void ExtendUserStatus(
+            string statusName,
+            int additionalSecs,
+            SuccessCallback success = null,
+            FailureCallback failure = null,
+            object cbObject = null)
+        {
+            Dictionary<string, object> details = new Dictionary<string,object>();
+            Dictionary<string, object> data = new Dictionary<string, object>();
+            data[OperationParam.PlayerStateServiceStatus.Value] = statusName;
+            data[OperationParam.PlayerStateServiceAdditionalSeconds.Value] =additionalSecs;
+            data[OperationParam.PlayerStateServiceDetails.Value] =details;
+
+            ServerCallback callback = BrainCloudClient.CreateServerCallback(success, failure, cbObject);
+            ServerCall sc = new ServerCall(ServiceName.PlayerState, ServiceOperation.ExtendUserStatus, data, callback);
+            _client.SendRequest(sc);
+        }
+
+        /// <summary>
+        /// Gets the Status.
+        /// </summary>
+        /// <remarks>
+        /// Service Name - PlayerState
+        /// Service Operation - GetUserStatus
+        /// </remarks>
+        /// <param name="statusName">
+        /// The name of the status.
+        /// <param name="success">
+        /// The success callback.
+        /// </param>
+        /// <param name="failure">
+        /// The failure callback.
+        /// </param>
+        /// <param name="cbObject">
+        /// The user object sent to the callback.
+        /// </param>
+        public void GetUserStatus(
+            string statusName,
+            SuccessCallback success = null,
+            FailureCallback failure = null,
+            object cbObject = null)
+        {
+            Dictionary<string, object> data = new Dictionary<string, object>();
+            data[OperationParam.PlayerStateServiceStatus.Value] = statusName;
+
+            ServerCallback callback = BrainCloudClient.CreateServerCallback(success, failure, cbObject);
+            ServerCall sc = new ServerCall(ServiceName.PlayerState, ServiceOperation.GetUserStatus, data, callback);
+            _client.SendRequest(sc);
+        }
+
+        /// <summary>
+        /// Sets the Status.
+        /// </summary>
+        /// <remarks>
+        /// Service Name - PlayerState
+        /// Service Operation - SetUserStatus
+        /// </remarks>
+        /// <param name="statusName">
+        /// The name of the status.
+        /// <param name="durationSecs">
+        /// The number of seconds to add.
+        /// <param name="details">
+        /// The details of the status.
+        /// <param name="success">
+        /// The success callback.
+        /// </param>
+        /// <param name="failure">
+        /// The failure callback.
+        /// </param>
+        /// <param name="cbObject">
+        /// The user object sent to the callback.
+        /// </param>
+        public void SetUserStatus(
```

Comment:
This is also done incorrectly, 
Refer to your good implementation of SubmitCrashReport and go from there